pkg/receive/handler.go

```diff
@@ -288,13 +373,30 @@ func (h *Handler) forward(ctx context.Context, tenant string, wreq *prompb.Write
 				return
 			}
 			req.Header.Add(h.options.TenantHeader, tenant)
+			req.Header.Add(h.options.ReplicaHeader, strconv.FormatUint(replicas[endpoint], 10))
+
+			// Increment the counters as necessary now that
+			// the requests will go out.
+			defer func() {
+				requestCounter.Inc()
```

Comment:
This will also be increased, even when there is an error. Is that intended?

Reply:
yes, this is tracking the _total_, number of requests. failed or successful. Using the error counter we can know the percentage of errors.

Reply:
it's probably worth unifying in one metric with different labels for success/failure

Reply:
Yep, I'd prefer having a unified counter. :+1:

Reply:
so don’t track total; instead use one counter vec with a result label for success or failure. When we want to know the percentage of failed request we do errors/(errors+successes)?

Reply:
correct